docs/guides/site-to-site-connectivity/end-customers.mdx

```diff
@@ -0,0 +1,245 @@
+
+ngrok has three primitives: the [**network**](/docs/network-edge/), the [**agent**](/docs/agent/), and the [**dashboard**](https://dashboard.ngrok.com/).
+
+- The network is a globally distributed infrastructure that accepts traffic to endpoints, applies policies, and routes connections to the appropriate agent.
+- The agent is a lightweight program with zero runtime dependencies, which forwards requests from your vendor to your upstream services and data.
+- The dashboard is a SaaS web app for configuring accounts, endpoints, and access policy. Depending on the architecture and arrangement with your vendor, you may or may not have access to an ngrok dashboard.
+
+In a generic site-to-site connectivity architecture, these pieces come together to do the following:
+
+1. The ngrok agent in your network connects to one or more of your upstream services.
+2. The ngrok agent connects out to the ngrok network at the default [agent ingress address](/docs/agent/ingress/) at `connect.ngrok-agent.com` (or a custom one).
+3. The ngrok network creates an endpoint like `your-company.your-vendor.com`, which then points to your upstream service.
+4. Your vendor sends requests from their services to the endpoint, which are then routed through to the agent and finally your upstream service.
+
+### What is ngrok’s network architecture?
+
+We run ngrok’s primary network and services on AWS.
+
+This architecture includes multiple [Points of Presence](/docs/network-edge/#points-of-presence) (PoPs) worldwide that run our services, route customer traffic, and make up our always-on [Global Server Load Balancer](https://ngrok.com/blog-post/gslb-global-server-load-balancing) (GSLB).
+
+The GSLB, which is active by default for all ngrok endpoints and tunnels, enhances the resiliency of the connection between your services and your vendor’s with DDoS protection and geo-failover that automatically routes traffic to the lowest-latency PoP in the event of downtime or service interruption. If you have specific data residency requirements, like those located in the EU, you and your vendor can configure ngrok to use a specific region in Europe or pin all traffic to a specific PoP.
+
+There are many possible architectures for setting up a site-to-site network based on the shared requirements of you and your vendor. Your vendor will work with you to find a mutually secure and reliable architecture.
+
+### How does ngrok handle data residency requests?
+
+ngrok is user-configurable to match your data residency needs.
+
+First, [configure the agent](/docs/guides/upgrade-v2-v3/#changes-to-choosing-a-region) to use a PoP in one of our [supported regions](/docs/network-edge/#points-of-presence). Next, work with your vendor to set up appropriate DNS to route all connections through the same data plane.
+
+Our regional data planes are located in Australia (Sydney), Europe (Frankfurt), India (Mumbai), Japan (Tokyo), South America (São Paulo), United States (California and Ohio).
```

Comment:
Our regional data planes are located in Australia (Sydney), Europe (Frankfurt), India (Mumbai), Japan (Tokyo), South America (São Paulo), and the United States (California and Ohio).